core/services/ocr2/plugins/ocr2keeper/util.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/services/job"
 	"github.com/smartcontractkit/chainlink/v2/core/services/keystore/keys/ethkey"
 	"github.com/smartcontractkit/chainlink/v2/core/services/ocr2/models"
-	kevm "github.com/smartcontractkit/chainlink/v2/core/services/ocr2/plugins/ocr2keeper/evm"
+	kevm "github.com/smartcontractkit/chainlink/v2/core/services/ocr2/plugins/ocr2keeper/evm20"
```

Comment:
this needs some more thought but since these utils are common across both versions, version can be sent as a parameter to EVMDependencies which chooses which dependencies to return. Will be more important when this is connected to the delegate (are you planning to do that in a separate PR?)